# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini
# Commit: bc3a104f8b20
# Full Hash: bc3a104f8b202e2501b6d0bee35ff60ab1d58733
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2019-12-21 09:45:32
# Regressor Bug: 1602410
# File Overlap Count: 3
# Description:
#   Bug 1602410. Make `window.print()` invoke printing via the parent process using a JSWindowActor. r=Gijs,kmag
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D56414
# ==============================================================================

diff -r 2e89a1d89e56 -r bc3a104f8b20 testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini
--- a/testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini	Sat Dec 21 00:46:20 2019 +0000
+++ b/testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini	Sat Dec 21 00:22:42 2019 +0000
@@ -1,4 +1,5 @@
 [print-microtask-after-navigate.html]
+  disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1569400#c22
   expected:
     if (os == "android") and not debug: ["ERROR", "OK"]
     if (os == "android") and debug: ERROR